Feat/configmap superseeds inline (#103)
traefik/traefikee-helm-chart: traefikee/templates/NOTES.txt

!!          This field may be removed in a future API version.                                                 !!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
{{ end }}

{{- if and .Values.controller.staticConfig.configMap .Values.controller.staticConfig.content }}
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

> !! WARNING: you declared both .spec.controller.staticConfig.configMap and .spec.controller.staticConfig        !!
> !!          .content.                                                                                          !!
> !!          Only .spec.controller.staticConfig.configMap will be taken into account.                           !!

Reviewer: ```suggestion
!! WARNING: .spec.controller.staticConfig.configMap and .spec.controller.staticConfig.content    !!
!!          has both been set.                                                                                                                !!
!!          /!\ Only .spec.controller.staticConfig.configMap will be taken into account.                         !!
```
